assets/in

```diff
@@ -133,6 +133,12 @@ git --no-pager log -1 --pretty=format:"%ae" > .git/committer
 # pulled ref in following tasks and resources.
 echo "${return_ref}" > .git/ref
 
+# Store commit message in .git/commit_message. Can be used to inform about
+# the content of a successfull build.
+# Using https://github.com/cloudfoundry-community/slack-notification-resource
+# for example
+echo $(git_metadata) | jq -r '.[6].value // ""' > .git/commit_message
```

Comment:
Rather than plucking it out of the metadata, I think it makes more sense to just get it the same way the metadata gets it (`git log -1 --format=format:%B`). That way it won't break if we add another metadata field before that one (`6`).